Makefile

```diff
@@ -236,7 +247,6 @@ release/gen-files: release/gen-templates
 
 .PHONY: release/image
 release/image:
-	bazel clean --expunge
```

Comment:
We put this in to stop us from using a stale UBI base image - how are we making sure that no longer happens?

Reply:
Ideally we shouldn't run this target manually. For rare cases, when we have to, I'd leave the clean step.

Reply:
The clean step kills development. We cannot have it here. If we need a release target I understand that.

Reply:
We can add this step to the release script then.

Reply:
@rail are we running the release locally or on a clean vm/test?  We may not have the cache during the release

Reply:
I reverted and I will add a build issue

Reply:
I reverted the change and added an issue to fix the container pull.